lib/angulartics-newrelic-insights.js

```diff
@@ -16,6 +16,7 @@
     .config(['$analyticsProvider', function($analyticsProvider) {
       angulartics.waitForVendorApi('newrelic', 100, function(newrelic) {
         $analyticsProvider.registerEventTrack(function(action, properties) {
+          properties.eventType = 'PageAction';
```

Comment:
I will make the changes. I am contacting new relic to see if I can get some official documentation about it.